tenable/io/v3/vm/target_groups.py

```diff
@@ -0,0 +1,142 @@
+"""
+Target Groups
+=============
+
+The following methods allow for interaction into the Tenable.io
+:devportal:`target_groups <target-groups>` API endpoints.
+
+Methods available on ``tio.target_groups``:
+
+.. rst-class:: hide-signature
+.. autoclass:: TargetGroupsAPI
+    :members:
+"""
+from typing import List, Dict
+
+from tenable.io.v3.base.endpoints.export import UWBaseEndpoint
+from tenable.io.v3.vm.schema import TargetGroupsSchema
+from tenable.utils import dict_merge
+
+
+class TargetGroupsAPI(UWBaseEndpoint):
+    _path = "target-groups"
+    _conv_json = True
+
+    def create(self, name: str, members: List, **kw) -> Dict:
+        """
+        Create a target-group.
+
+        :devportal:`target-groups: create <target-groups-create>`
+
+        Args:
+            name (str): The name of the target group
+            members (list):
+                The members of the target group.  FQDNs, CIDRs, IPRanges, and
+                individual IPs are supported.
+            acls (list, optional):
+                A list of ACLs defining how the asset list can be used.  For
+                further information on how the ACL dictionaries should be
+                written, please refer to the API documentation.
+
+        Returns:
+            Dict:
+                The resource record of the newly created target group.
+
+        Examples:
+            >>> tg = tio.target_groups.create('Example', ['192.168.0.0/24'])
+        """
+        schema = TargetGroupsSchema()
+        if not members:
+            members = []
+        members = ",".join(members)
+        payload = schema.load({"name": name, "members": members, **kw})
```

Comment:
schema.dump() and schema.load() gives same object when given input is dictionary. So
schema.load() instead of schema.dump(schema.load()) performs 2x better

Reply:
Updated